HW0/P5/P5.txt

```diff
@@ -1 +1,49 @@
 # Your answers here.
+
+1) Need to do 255 summations - 255 seconds
+
+2) Yes, the 8 of them can split the bags evenly and find the sum of their respective 32 bags
+which would take 31 seconds. 4 of them then take the sums of the other 4 people and sum their
+numbers together (1 second), then 2 take the sum of the other 2 (1 second), then 1 person 
+takes the final sum (1 second). So in total 34 seconds + whatever time it takes to communicate
+the sums between employees.
+
+3) Using the same process of adding numbers, then taking half the employees to add the remaining numbers...
+	128 employees sum 2 numbers - 1 second
+	64 employees sum 2 numbers - 1 second
+	...
+	1 employee sums 2 numbers
+
+	For a total of 8 seconds
+
+4) ceil(log_2(N)) seconds.
```

Comment:
I agree the answer is ceil(log_2(N)) seconds, but this can be hardly seen on your corresponding plot. Why not narrow the range of x, or better, compute log for y axis